Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -291,8 +291,11 @@
 
     private static boolean isLogging_ = false;
 
-    private static boolean checkInstallReferrer_ = false;
-    private static long PLAYSTORE_REFERRAL_FETCH_WAIT_FOR = 5000;
+    // Branch will wait for 1.5 sec to capture the install referrer inorder to provide
+    // accurate attribution . This will be only on the first time user open the app.
+    // @see to skip this please call {@link #disablePlayStoreReferrer() }
+    private static boolean checkInstallReferrer_ = true;
```

Comment:
I'm conflicted as well @Sarkar -- it's probably one of those things devs/users are never going to notice, but as a performance queen I would not want *any* SDK by default taking 1.5 seconds extra to gather something that 1) might not show up in time (1.5 seconds seems like it won't?) 2) I might not care about.